CONTRIBUTING.md

```diff
@@ -425,7 +425,19 @@ union my_array_64bits
 3. Variable and function naming should use “snake case”, namely descriptive names which are all lowercase and separated by an underscore
    1. `int32_t this_is_an_example_of_a_variable;`
 
-4. For loops shall have the following form:
+4. `if` statements shall have the following form:
+
+```c
+int32_t sum;
+const static int32_t max = 25;
+
+if ( sum > max )
+{
+    sum = max;
+}
+```
+
+4. `for` loops shall have the following form:

```

Comment:
Check your numbering.  To prevent breaking your numbered lists when using code blocks, you have to indent the code blocks by 2 or 4 spaces.